Applications/FileIO/XmlIO/Qt/XmlPrjInterface.cpp

```diff
@@ -319,15 +310,13 @@ bool XmlPrjInterface::write()
     }
 
     // stations
-    std::vector<std::string> stnNames;
-    geo_objects.getStationVectorNames(stnNames);
-    for (std::vector<std::string>::const_iterator it(stnNames.begin());
-         it != stnNames.end();
-         ++it)
+    std::vector<std::string> stn_names;
+    geo_objects.getStationVectorNames(stn_names);
+    for (std::string const name : stn_names)
     {
         // write station file
         GeoLib::IO::XmlStnInterface stn(geo_objects);
-        std::string name(*it);
+        std::string name(name);
```

Comment:
I'd guess this line is not needed. If the non-const version of `name` is needed, remove the const in the for-loop.